include/internal/catch_default_main.hpp

```diff
@@ -15,7 +15,7 @@
 
 #if defined(CATCH_CONFIG_WCHAR) && defined(CATCH_PLATFORM_WINDOWS) && defined(_UNICODE) && !defined(DO_NOT_USE_WMAIN)
 // Standard C/C++ Win32 Unicode wmain entry point
-extern "C" int wmain (int argc, wchar_t * argv[], wchar_t * []) {
+extern "C" int __cdecl wmain (int argc, wchar_t * argv[], wchar_t * []) {
```

Comment:
Are there compilers that support wmain but don't have __cdecl? If so, we'd need to chain to another define, such as:

```c++
#ifndef CDECL
#ifdef MSVC
#define CDECL __cdecl
#else
#define CDECL
#endif
#endif
```

Reply:
I don't think there are any.